### DISCUSSION TITLE
build-storybook with --profile option

### DISCUSSION THREAD

**Discussion by CodeguruEdison** [storybookjs/storybook#13611] [Help]
Do we know, how we can build storybook with profile option. In My application I created React Hoc for React profiler component. As per react documentation, we need to build with --profile option to make profiler work as application built with CRA. Just wondering is it possible to do same with storybook build option or any work around to achieve this?

**Comment by shilman** [comment]
i don't have an answer for you, but this addon might be of interest https://storybook.js.org/addons/storybook-addon-performance
